lib/navigation/backoffice_navigator.dart

```diff
@@ -124,37 +118,59 @@ class BackofficeNavigator extends CommonNavigator {
         ),
       );
     },
+    routes: [settings],
```

Comment:
I don't see the settings page definition. The two settings sub pages are not added.

Reply:
Yes it is defined, you don't see it on this PR because it was not modified as it already worked like this. 
You can find it at line 100 of the backoffice_navigator